config/default.yml

```diff
@@ -213,7 +213,10 @@ RSpec/ExampleWording:
 RSpec/ExpectChange:
   EnforcedStyle: block
 
-RSpec/FilePath:
+RSpec/SpecFilePathSuffix:
+  Enabled: false
```

Comment:
can we enable this one?

Reply:
I wasn't sure if that would work well for us without some tweaks to consuming applications, because I think it also catches things like `spec/spec_helper.rb`. So we'd need to figure out all the exclude paths we'd want to add.

Probably not too hard to do in a separate PR, but right now CI is broken for certain repos, so I want to unblock that. If I understand correctly, both of them false is equivalent behavior to today.

Reply:
Yeah, here's an example of the rule in the wild, and it requires excluding all non spec files in your spec folder https://github.com/bkuhlmann/gemsmith/blob/main/.rubocop.yml